tasks/push-snapshot/push-snapshot.yaml

```diff
@@ -97,20 +92,33 @@ spec:
           repository=$(jq -r '.repository' <<< $component)
           imageTags=$(jq '.tags' <<< $component)
 
-          arches=$(get-image-architectures "${containerImage}" | jq -s 'map(.platform.architecture)')
+          # oras has very limited support for selecting the right auth entry,
+          # so create a custom auth file with just one entry.
+          AUTH_FILE=$(mktemp)
+          select-oci-auth "${containerImage}" > "$AUTH_FILE"
+
           # Just read the first from the list of architectures
```

Comment:
nit picky but this comment makes a lot more sense before lines 105 and 106 instead of when we fetch the entire arrays

Reply:
moved comment